server/routes/index.js

```diff
@@ -23,28 +23,28 @@ router.delete('/entries/:id', Middleware.isLoggedIn, Middleware.validateParams,
 router.get('/entries', Middleware.isLoggedIn, EntryController.getAllEntries);
 router.get('/entries/:id', Middleware.isLoggedIn, Middleware.validateParams, EntryController.getEntry);
 
-router.get('*', (req, res, next) => {
+router.get('*', (req, res) => {
   res.status(404).json({
     status: 'error',
     message: '404 Page not found'
   });
 });
 
-router.post('*', (req, res, next) => {
+router.post('*', (req, res) => {
   res.status(404).json({
     status: 'error',
     message: '404 Page not found'
   });
 });
 
-router.put('*', (req, res, next) => {
+router.put('*', (req, res) => {
   res.status(404).json({
     status: 'error',
     message: '404 Page not found'
   });
```

Comment:
you can have a `router.all('/*', (req, res) => { });` instead of repeating for `get, put, post etc...`